al2.pkr.hcl

```diff
@@ -174,6 +174,10 @@ build {
     script = "scripts/enable-services.sh"
   }
 
+  provisioner "shell" {
```

Comment:
I see this only for al2. Is it not needed for al2022?

Reply:
We don't have rpm available for al2022 yet. I will update the description as well, thanks for calling it out.